src/components/ec-panel/ec-panel.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <aside
+    v-if="showPanel"
+    class="ec-panel"
+  >
+    <header class="ec-panel__header">
+      <div class="ec-panel__header-icons">
+        <ec-icon
+          class="ec-panel__close"
+          name="simple-close"
+          :size="24"
+          @click="$emit('close')"
+        />
+      </div>
+
+      <div>
+        <slot name="header" />
+      </div>
+    </header>
+
+    <main>
+      <slot name="main" />
+    </main>
+  </aside>
+</template>
+
+<script>
+import EcIcon from '../ec-icon';
+
+
+export default {
+  name: 'EcPanel',
+  components: {
+    EcIcon,
+  },
+  model: {
+    prop: 'showPanel',
+    event: 'close',
+  },
+  props: {
+    showPanel: {
+      type: Boolean,
+      default: false,
+    },
+  },
+
+};
+</script>
+
+<style lang="scss">
+@import '../../scss/settings/colors/index';
+@import '../../scss/tools/index';
+
+$ec-panel-close-btn-fill: $level-4-interactive-elements !default;
+$ec-panel-close-btn-fill-hover: $level-4-tech-blue !default;
+
+.ec-panel {
+  display: block;
+  width: 100%;
+  max-width: 352px;
+  height: 100vh;
+  min-height: 100%;
+  position: absolute;
+  background: white;
```

Comment:
hard-coded color name

Reply:
+1, however personally I prefer not to extract white and black color names and always hardcode `#fff` and `#000` white will always be white, and black will always be black

Reply:
I saw we already added white as a var so I went for "$ec-panel-background-color: $white !default"

Reply:
I think these colors should be in variables, but I don't like the names `$white` and `$black`. White might not be white everywhere, imagine we have light and dark theme. How the code would look like? 
```
// light.scss
$black: #000;
$white: #fff;

// dark.scss
$black: #fff;
$white: #000;
```

We have nice names for `error`, `disabled`, and `interactive-elements`. Why do we treat white as `$white`?